docs/src/installation/FreeBSD.md

```diff
@@ -14,24 +14,9 @@ This guide will help you to install `spotifyd` on FreeBSD and have it always run
 sudo pkg install spotifyd
 ```
 
-## Config file
-
-
-Input the following data. Change the **username**, **password**, **device_name**, and the **bitrate**.
-
-```toml
-[global]
-username = "USER"
-password = "PASS"
-backend = "portaudio"
-device = "/dev/dsp"
```

Comment:
maybe keep these 2 settings? i think the are required for freebsd but not sure

Reply:
Should've read what I was removing more carefully :sweat_smile:

I just fired up a FreeBSD-VM and tested spotifyd there without any configuration (except for username and password). It did indeed not work since spotifyd is compiled there [with only portaudio enabled](https://codeberg.org/FreeBSD/freebsd-ports/src/commit/775d857db4e7eb90ca78745da1e7f15fc8f2e50b/audio/spotifyd/Makefile#L419), but the fallback in spotifyd [is always alsa](https://github.com/Spotifyd/spotifyd/blob/ae6dac7a54f899316674ba57ce4a0f9890cd2b1c/src/config.rs#L604-L608) (which should probably be fixed to be more feature-agnostic). However, after specifying `backend = "portaudio"` everything worked fine, so the `device` is apparently picked correctly by default.

So I guess, I'll add a mention about the backend and we can maybe remove that after the fallback has been fixed.

Reply:
I readded the backend thing, do you think that we can omit the device part or should I mention that too?

@rosorio Maybe you can have a look at that paragraph as well, since it was you who originally added that section?

Reply:
I can confirm that these are needed, as it's what I'm using on my daily driver T480s running FreeBSD and Wayland+Sway.

Additionally [portaudio is the default for the FreeBSD port/package](https://www.freshports.org/audio/spotifyd#config), because portaudio maintains the same OSS compatibility that FreeBSD still has (and has had since Linux had OSS compatibility).

Reply:
@debdrup Thank you for the feedback! I will also add the device value back to the documentation.

Would you mind sharing what `spotifyd` does / why it doesn't work, when running without the `device` parameter? (Potentially in a new issue so that it is easier to keep track of that.) Because the ideal behaviour is of course that `spotifyd` just starts on most setups without any configuration needed. For the `backend` config, I'm already quite sure what is causing this, but the `device` config is not required in my VM, so I can't test this. Thank you!

Reply:
> maybe keep these 2 settings? i think the are required for freebsd but not sure

Backend and device settings are required for FreeBSD environment.
Currently spotifyd is build to only support portaudio backend.

Reply:
I think I phrased myself poorly, because the device parameter isn't required on FreeBSD, unless you want to play on a different pair of speakers (ie. `/dev/pcm1` if `/dev/pcm0` is the default).

Reply:
@rosorio Just to make sure: If you omit the `device` parameter from the config (but leave the `backend = "portaudio"`), does it really stop working and, if so, could you open an issue with details about potential errors? Otherwise, I would leave it as it is now (see [here](https://github.com/Spotifyd/spotifyd/blob/f075168162dd221cf79bc1cbf940215684e20c69/docs/src/installation/FreeBSD.md?plain=1#L19) for the current version, which just mentions the `backend` thing). Thank you both!

Reply:
> I readded the backend thing, do you think that we can omit the device part or should I mention that too?
> 
> @rosorio Maybe you can have a look at that paragraph as well, since it was you who originally added that section?



> I readded the backend thing, do you think that we can omit the device part or should I mention that too?
> 
> @rosorio Maybe you can have a look at that paragraph as well, since it was you who originally added that section?

Thanks @eladyn, you have my +1 to push your changes.